platform/wisdomctl/wisdomctl.py

```diff
@@ -0,0 +1,31 @@
+from ocifs import OCIFileSystem
+
+import click
+
+@click.group()
+def cli(): pass
+
+REMOTE = click.argument("remote")
+
+MB = 1024 ** 2
+def slowCopy(src, dest):
+    while True:
+        hunk = src.read(MB)
+        if not hunk: break
+        dest.write(hunk)
+
+@cli.command()
+@REMOTE
```

Comment:
the interpolation of 8 into 18 and 26 may slow down future time travelers reading the code base. Why not create two copies of 8, and replace 18 and 26?

Software engineering practices such as DRY are good **guidelines** but they are not mandatory **rules**. I see Dry violated all the time for the right reasons, and all the time for the wrong reasons.

Reply:
In practice, these arguments will be used many times, and eventually we will desire deduplication merely so that we accept the same arguments in different commands. For now, I've reduplicated the decorator as asked.